From b38ae24985e90329a68bd26718da52ed4ecfab9d Mon Sep 17 00:00:00 2001
From: Srinivas Pandruvada <srinivas.pandruvada@linux.intel.com>
Date: Mon, 3 Feb 2014 08:12:26 -0800
Subject: [PATCH 1/3] Remove warning for non Intel processor

Thermal daemon can still work with limited capability for non Intel
processors, so remove check and warning.
---
 src/thd_engine.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/thd_engine.cpp b/src/thd_engine.cpp
index d1ceba5..33d8068 100644
--- a/src/thd_engine.cpp
+++ b/src/thd_engine.cpp
@@ -562,7 +562,8 @@ int cthd_engine::check_cpu_id() {
 	if (ebx == 0x756e6547 && edx == 0x49656e69 && ecx == 0x6c65746e)
 		genuine_intel = 1;
 	if (genuine_intel == 0) {
-		thd_log_warn("Not running on a genuine Intel CPU!\n");
+		// Simply return without further capability check
+		return THD_SUCCESS;
 	}
 	asm("cpuid": "=a"(fms), "=c"(ecx), "=d"(edx): "a"(1): "ebx");
 	family = (fms >> 8) & 0xf;
-- 
1.9.rc1

